### PR TITLE
Feature/pylama warns

### PR DESCRIPTION
**Pull request by bhirsz**
Fix pylama reported issues (and ignore not important ones)
Refactor importing modules to use one common method instead

I had to ignore whole docs in pylama since some files need to use specific format

### CODE REVIEW

File: robocop/utils/file_types.py [hunk 1]
```diff
@@ -23,6 +26,10 @@ def get_parser(self):
 
 
 class FileTypeChecker(ast.NodeVisitor):
+    """
+    Check if file contain import statements. If the import is in list of files to be scanned, update its type
```
Comment (mnojek) [inline robocop/utils/file_types.py:30]: Typo: contains

File: robocop/run.py [hunk 2]
```diff
@@ -23,17 +26,25 @@ def __init__(self):
...
 
     def run(self):
+        """ Entry point for running scans """
         self.recognize_file_types()
         self.run_checks()
         self.make_reports()
 
     def recognize_file_types(self):
+        """
+        Preparse files to recognize types. If the filename is "__init__.robot" file type is INIT.
```
Comment (mnojek) [inline robocop/run.py:44]: Typo: Prepare

File: robocop/run.py [hunk 3]
```diff
@@ -23,17 +26,25 @@ def __init__(self):
...
+        If the file is imported somewhere then file type is RESOURCE. Otherwise file type is GENERAL.
+        Those types are important since are used to define parsing class for robot API.
```
Comment (mnojek) [inline robocop/run.py:46]: I would rephrase it to:
```suggestion
        Prepare files to recognize their types. If the filename is `__init__.robot`, the type is `INIT`.
        If the file is imported somewhere then file type is `RESOURCE`. Otherwise file type is `GENERAL`.
        These types are important since they are used to define parsing class for robot API.
```

Comment (bhirsz) [reply robocop/run.py:46]: it's actually "preparse" as we're parsing files (in limited way) to detect file type. So what could be best name?

Comment (mnojek) [reply robocop/run.py:46]: Oh, ok 😄 I updated the suggestion.
